docs/documentation/cert-manager.md

```diff
@@ -6,7 +6,7 @@ permalink: /docs/cert-manager
 ---
 ### Cert-Manager
 
-Dekorate supports generating `Certificate` and `Issuer` resources using [Cert-Manager](https://cert-manager.io/). When these resources are loaded into the cluster, Cert-Manager will issue the certificates, issuers, and trust-store files that then can be used by final applications.
+Dekorate supports to generate a X.509 certificate with the help of the Certificate and Issuer CRD resources handled by the [Cert-Manager](https://cert-manager.io/). When these CRD resources are deployed on the cluster, the Cert-Manager will process them in order to populate a Secret containing by example a: CA certificate, private key, server certificate or java keystores, etc.
```

Comment:
Can you also add a hyperlink to point to this section of the doc https://cert-manager.io/docs/configuration/selfsigned/ which explains very well the different concepts (issuer, certificate, ...). If you do that, then review the last sentence with something like "... process them in order to populate for a Selfsigned certificate a Secret containing: xxxx". Replace xxxx with what the secret will exactly include

Reply:
The guide in the link you shared explains how to use self-signed certificates only and at this point, I only wanted to give a short introduction to the Cert-Manager concepts. 
The link about how to use a self-signed certificate is used later in the guide.

Reply:
Yes but as the certificate manager documentation is not great at all, this is from what I know one of the best documented place where they cover what an issuer and certificate CRD are ;-)

Reply:
I've updated the issuer types section to include more examples. Let me know if it looks better to you.